force-app/main/default/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.js

```diff
@@ -240,6 +269,16 @@ export default class rd2EntryFormScheduleSection extends LightningElement {
      * @param event
      */
     handleAdvancedPeriodChange(event) {
+
+        if(this.isExperienceSite) {
+            if((event.target.value === MONTHLY && this.isElevateDonation && this.isInitiallyMonthlyDonation)||!(event.target.value === MONTHLY)) {
```

Comment:
Can we extract this guard clause into its own function? it seems complex enough for us to be able to name what it does exactly.

Reply:
Ok, I'll move it to a new function.